src/domain/event/eventInfo/__tests__/EventInfo.test.tsx

```diff
@@ -364,3 +391,225 @@ describe('OrganizationInfo', () => {
     }
   );
 });
+
+describe('superEvent', () => {
+  it('should render super event title and link when super event is given', async () => {
+    const superEvent = fakeEvent({
+      superEvent: { internalId: superEventInternalId },
+    });
+    const superEventResponse = {
+      data: superEvent,
+      status: 'resolved',
+    } as SuperEventResponse;
+    const { history } = render(
+      <EventInfo
+        event={event}
+        superEvent={superEventResponse}
+        eventType="event"
+      />,
+      {
+        mocks,
+      }
+    );
+    await actWait();
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.superEvent.title,
+      })
+    ).toBeInTheDocument();
+
+    userEvent.click(
+      within(screen.getByTestId(superEventTestId)).queryByText(
+        superEvent.name.fi
+      )
+    );
+    expect(history.location.pathname).toBe(`/fi/events/${superEvent.id}`);
+  });
+
+  it('should should not render super event title when super event is not given', async () => {
+    render(<EventInfo event={event} eventType="event" />, {
+      mocks,
+    });
+    await actWait();
+
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.superEvent.title,
+      })
+    ).not.toBeInTheDocument();
+  });
+});
+
+describe('subEvents', () => {
+  const meta: Meta = {
+    count: 20,
+    next:
+      // eslint-disable-next-line max-len
+      'https://api.hel.fi/linkedevents/v1/event/?include=keyword,location&page=2&sort=start_time&start=2020-08-11T03&super_event=hel:123',
+    previous: null,
+    __typename: 'Meta',
+  };
+
+  const subEventsResponse = {
+    ...fakeEvents(
+      10,
+      range(1, 11).map((i) => ({
+        endTime: addDays(new Date(endTime), i).toISOString(),
+        startTime: addDays(new Date(startTime), i).toISOString(),
+        typeId: i % 2 === 0 ? EventTypeId.Course : EventTypeId.General,
+        superEvent: { internalId: superEventInternalId },
+      }))
+    ),
+    meta,
+  };
```

Comment:
Moved them to utils -folder. Couldn't use them from __mocks__ -forlder because of this : "imported from a __mocks__ directory. Instead use `jest.mock` and import from the original module path jest/no-mocks-import"